### PR TITLE
Capstone Project: StartUp Lab Summit 2021

### PR DESCRIPTION
**Pull request by faizi2500**
## This Pull Request contains the Capstone Project for Module 1 ##

This projects follows a template provided to us on [Behance by Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

The following things have been accomplished in this task:

- [x] Home Page (desktop and mobile)
- [x] About Page (desktop and mobile)
- [x] Mobile Menu
- [x] Dynamically displayed Guests section using JavaScript.

### According to Microverse's requirements, the following criteria were fulfilled. ###
- [x] The home page should have a link in the menu to the about page.
- [x] The logo in the header links to the home page.
- [x] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- [x] The mobile menu should have a close (X) button that closes the menu.
- [x] The section "Featured speakers" should be created dynamically in JavaScript.

[CLICK HERE FOR DEMO](https://www.loom.com/share/44ab3ae3582e4eb6aa25ce0dfff240b5)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
```
Comment (mricanho) [inline README.md]: - [ ] Add an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        <h4>English</h4>
+      </li>
+      <li>
+        <h4>My Page</h4>
+      </li>
+      <li>
+        <h4>Logout</h4>
+      </li>
+    </ul>
+    </div>
```
Comment (mricanho) [inline about.html:38]: - [ ] Reduce the margins between these elements to resemble the original design

- [ ] Change the black background to the correct grey color

Your design:

imagen

Original design:

imagen

File: about.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      <img class="partner-image" src="assets/partner3.png" alt=""/>
+    </div>
+    <div class="partners-list">
+      <img class="partner-image" src="assets/partner4.png" alt=""/>
+    </div>
+    <div class="partners-list">
+      <img class="partner-image" src="assets/partner5.png" alt=""/>
+    </div>
+  </div>
+  </section> -->
```
Comment (mricanho) [inline about.html]: - [ ] Delete all the commented code, remember that a clean code it's a happy code

File: index.html [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+            Join StartUp Lab Summit 2021
+          </button>
+        </div>
+      </div>
+    </section>
+    <div class="section-speakers-heading">
+      <h2>Featured Speakers</h2>
+      <hr />
+    </div>
+    <!-- <section id="speaker-container"></section> -->
```
Comment (mricanho) [inline index.html]: - [ ] Unnecessary commented code here!

File: js/data.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+const data = [
```
Comment (mricanho) [inline js/data.js:1]: - Good work with this code

File: js/section.js [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    id: 6,
+    pic: './assets/image-6.png',
+    altText: 'This is featured speaker image',
+    nameOfSpeaker: 'Toni',
+    designation: 'Shopify Founder',
+    history: 'He is a venture capitalist, engineer, SPAC sponsor and the founder and CEO of Social Capital. Palihapitiya was an early senior executive at Facebook, working at the company from 2007 to 2011.',
+  },
+];
+
+const speakerContainer = document.createElement('section');
```
Comment (mricanho) [inline js/section.js:52]: - Well structured code, nicely done

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+package.json
```
Comment (mricanho) [inline .gitignore:3]: - [ ] [OPTIONAL]These two files can be in the root repository

File: home.css [hunk 8]
```diff
@@ -0,0 +1,654 @@
...
+  font-size: 16px;
+  font-family: "lato", sans-serif;
+}
+
+p {
+  line-height: 1.5rem;
+}
+
+.menu-content {
+  display: none;
```
Comment (mricanho) [inline home.css:23]: - [ ] You are repeating a lot of classes here, so in the spirit of clean code, group all of them, e.g.

```
.menu-content, .top-bar-section, .close-menu, .logo-destop a {
  display: none
}
```

File: index.html [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+          <button type="submit" class="program-button">
+            Join StartUp Lab Summit 2021
+          </button>
+        </div>
+      </div>
+    </section>
+    <div class="section-speakers-heading">
+      <h2>Featured Speakers</h2>
+      <hr />
+    </div>
```
Comment (mricanho) [inline index.html:164]: - [ ] In this section, you have a design problem, in the original design every single one of the speakers have their own box and the letters respect the top limit, in your design the letters exceed the top limit of the box, please fix it.

Your design:

Desktop
imagen

Mobile
imagen

Original design:

Desktop
imagen

Mobile
imagen

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @faizi2500, 

It's a very good first attempt! You are following the gitflow, your commit messages are excellent and your README file it's almost perfect, like your JS code, excellent job! 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

you can do it

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: page.css [hunk 10]
```diff
@@ -0,0 +1,506 @@
...
+}
+
+.pages-menu {
+  width: 100%;
+  display: none;
+  height: 80vh;
+  background-color: rgb(234, 234, 234, 0.7);
+  border-right: 4px outset #ec5242;
+  border-bottom: 2px outset #ec5242;
+}
```
Comment (mricanho) [inline page.css]: - [ ] There is a problem with the height, and it's an it's the height and width of the border line, you can fix this with a height: 100vh and a width: 100%

Your design problem:

imagen

**Comment by faizi2500** [comment]
Thank you for sharing such a detailed feedback. I have updated my files according to your suggestion and website looks very nice now.  Just left with few things and then Ill submit it for another review .
Thanks again 🙂

File: index.html [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+            the trade, offer advice, tips and tools for aspiring entrepreneurs
+            as they launch their business.
+          </p>
+        </div>
+        <div class="event-details">
+          <h3>2021.10.4(MON) ~ 6(WED)</h3>
+          <p class="venue">
+            @ Pearl Continental(PC) Hotel Bhurban, near Islamabad
+          </p>
+          <!-- <p><a href="#">Click here</a> to participate online.</p> -->
```
Comment (ElijahTobs) [inline index.html]: - [ ] As the previous reviewer mentioned, delete all commented code.

File: home.css [hunk 12]
```diff
@@ -0,0 +1,690 @@
...
+}
+
+.speaker-image {
+  width: 128px;
+  height: 128px;
+
+  /* align-self: center;
+  justify-self: center;
+  margin-block: auto;
+  z-index: 2; */
```
Comment (ElijahTobs) [inline home.css]: - [ ] You should also delete this commented block of code and on lines `570 and 626`

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @faizi2500  👋

You've done really well so far! 👍 

## Major Highlights 💯 
Detailed PR description ✔️ 
You've followed the Github flow rules. ✔️ 
All linter checks are passing ✔️ 
You have good commit messages ✔️ 


_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

___N/A___

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**